### PR TITLE
Nik john/add venue capacity

### PR DESCRIPTION
**Pull request by nik-john**
https://github.com/freeCodeCamp/chapter/issues/17

> @QuincyLarson Looks like the Event table will need 'capacity' to be added

@QuincyLarson Could you please take a look?

### CODE REVIEW

File: data/ddl.sql [hunk 1]
```diff
@@ -60,6 +60,7 @@ create table events (
     venue_id uuid references venues(id),
     tag_id uuid references tags(id),
     canceled boolean default false
```
Comment (ScottBrenner) [inline data/ddl.sql]:
```suggestion
    canceled boolean default false,
```

**Comment by vaibhavsingh97** [comment]
@nik-john Thanks for the PR, just wanna know why commits are mixed? Can you fix it or cherry-pick your commit and open a new PR

**Comment by nik-john** [comment]
@vaibhavsingh97 Yeah - I'll raise a new PR. Was in too much of a hurry 🤕

**Comment by nik-john** [comment]
https://github.com/freeCodeCamp/chapter/pull/63

**Comment by vaibhavsingh97** [comment]
@nik-john oh! not an issue. Please take your time 😇

**Pull request closed without merging**